src/components/Search/Search.tsx

```diff
@@ -0,0 +1,116 @@
+import * as React from 'react';
+import './Search.scss';
+import {Component} from "react";
+import {inject, observer} from "mobx-react";
+import * as Autosuggest from  'react-autosuggest';
+import {Fetch} from "../../helpers/fetch";
+import {env} from "../../env/environment";
+import {ITweet} from "../../interfaces";
+
+interface ISearchProps {
+  stores?: any;
+}
+
+@inject('stores')
+@observer
+export class Search extends Component<ISearchProps> {
+  constructor(props: ISearchProps) {
+    super(props);
+
+    this.state = {
```

Comment:
If you use State please create interface for it

Reply:
It is not good practice to mix mobx with state.